app/Db/pu610k.db

```diff
@@ -66,7 +80,8 @@ record(calc, "$(BASE):$(CHAN):MODE_ALLOW") {
     field(DESC, "PU610K Operating Mode")
     field(INPA, "$(BASE):$(CHAN):MODE")
     field(INPB, "$(BASE):$(CHAN):ENABLE_RBV")
-    field(CALC, "((A==0)||(B!=0))&&(A>=0)&&(A<=2)") 
+    field(INPC, "$(BASE):CHILLER_OK")
```

Comment:
moving `CODE_INTERLOCK` and `CHILLER_OK` into one record raises the question of whether or not mode allow should check chiller status before setting a mode.

Reply:
I think it is okay to bypass the check here. The user has to put the PFN into standby before charging. In that second `CHILLER_INTERLOCK` will scan and set the `MODE` to stop.